packages/pn-commons/src/components/PnInfoCard.tsx

```diff
@@ -0,0 +1,183 @@
+import { MouseEvent, ReactNode, useEffect, useState } from 'react';
+
+import KeyboardArrowDownOutlinedIcon from '@mui/icons-material/KeyboardArrowDownOutlined';
+import KeyboardArrowUpOutlinedIcon from '@mui/icons-material/KeyboardArrowUpOutlined';
+import MoreVert from '@mui/icons-material/MoreVert';
+import {
+  Box,
+  Card,
+  CardContent,
+  CardHeader,
+  CardProps,
+  IconButton,
+  Menu,
+  MenuItem,
+  Stack,
+  Typography,
+  TypographyProps,
+} from '@mui/material';
+import { useIsMobile } from '@pagopa-pn/pn-commons';
+
+type Props = {
+  title: ReactNode;
+  subtitle?: ReactNode;
+  actions?: Array<ReactNode>;
+  expanded?: boolean;
+  slotProps?: CardProps;
+  children: ReactNode;
+};
+
+const PnInfoCardHeading: React.FC<{
```

Comment:
use React.FC<TypographyProps>

Reply:
I didn't understand, we are already using React.FC here.